awkward/array/jagged.py

```diff
@@ -1350,9 +1362,42 @@ def concatenate(isclassmethod, cls_or_self, arrays):
 
         return cls(starts, stops, content)
 
-    @classmethod
-    def stack(cls, first, *rest):    # each item in first followed by second, etc.
```

Comment:
Now that I see this old stub here, I see why you called the new function `stack`. It is very likely that I made the same mistake, and that's why the stub exists...

Reply:
I'm either getting rid of stack or got rid of stack. It should be gone.